test-suite/coq-makefile/template/init.sh

```diff
@@ -4,16 +4,18 @@ set -o pipefail
 export PATH=$COQBIN:$PATH
 export LC_ALL=C
 
-rm -rf theories src Makefile Makefile.conf tmp
-git clean -dfx || true
+rm -rf _test
+mkdir _test
+find . -maxdepth 1 -not -name . -not -name _test -exec cp -r '{}' -t _test ';'
```

Comment:
With `extglob` this line could be `cp -r !(_test) _test`, maybe that would be better.
OTOH `bash -n` doesn't use extglob by default, you have to pass it `-O extglob` or it will complain. This is an annoyance when doing automatic syntax checking such as fly{check,make} in emacs.